.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Our [Code of Conduct](https://www.classicpress.net/democracy/#democracy-conduct) and [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions)
+Our code of conduct is simple: Be good to one another, take criticism well and follow the democratic process. When contributing to this repository, please note we have a [code of conduct](https://www.classicpress.net/democracy/#democracy-conduct) with some [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions). 
+
+Our request is that you follow these guidelines in all your interactions with the project. 
+
+# Start helping
+You can start helping by submitting pull requests(PRs) to the develop branch. This is not limited to:
+
+## Documentation
+Good documentation will make the work of making ClassicPress a better software much faster and a better user experience.
+
+## Fixing Bugs
+Most of the bugs are shared via the [issues page](https://github.com/ClassicPress/ClassicPress/issues). When evaluating bug fixes pull requests(PRs), we look for, if not all these things
+
+  - The change impacts existing ClassicPress users. (Otherwise, there are literally thousands of things we could look at, but we need to prioritize our development time.)
+  - The change is not going to break any other use cases.
```

Comment:
What kind of use cases? Like also backward compatibility?
This depends on the branch 1/2 about the focus of the change itself.

Reply:
These bullet points should apply to all types of changes. If we do a split (a branch 1 or 2 as you say) in this document, then it should be between major changes (which need a petition) and minor changes (which should usually be discussed beforehand, but we accept PRs as long as they meet these criteria).

Reply:
So better specify it